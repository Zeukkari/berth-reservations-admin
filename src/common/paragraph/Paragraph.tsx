import React from 'react';

import styles from './paragraph.module.scss';

const Paragraph = ({ children }: { children?: React.ReactNode }) => (
  <article className={styles.paragraph}>{children}</article>
);

export default Paragraph;
