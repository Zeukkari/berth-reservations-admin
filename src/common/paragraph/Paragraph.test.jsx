import React from 'react';
import { shallow } from 'enzyme';

import Paragraph from './Paragraph';

describe('Paragraph', () => {
  it('renders normally', () => {
    const getWrapper = () => shallow(<Paragraph>Content</Paragraph>);
    const wrapper = getWrapper();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
