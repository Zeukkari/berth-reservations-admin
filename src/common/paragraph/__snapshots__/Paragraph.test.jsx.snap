// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph renders normally 1`] = `"<article class=\\"paragraph\\">Content</article>"`;
